Player Display Update

Marthijn-B opened the pull request:
With new data.
Wonder cards placed on board
Cards in hand resized to one line

Marthijn-B committed:
Player Display Update

With new data.
Wonder cards placed on board
Cards in hand resized to one line

mmdonaldson reviewed (changes requested):
There are alot of code which has been deleted from my previous PR. Can you please rebase this with master

Marthijn-B committed:
re-based and updated

rebased Gamescreen.j back to MD previous PR
altered PlayerDisplay to handle new returned object from server

mmdonaldson commented:
@Marthijn-B hey mate the PR still deletes the stuff I built about play card error which is supposed to show if you have already played a card in the current round

Marthijn-B committed:
Playcard error 

This file has the playCard error in it

Marthijn-B committed:
Add files via upload

Marthijn-B committed:
Seperated Wonder Card component

Component to show wonder card under the board
